keras/applications/inception_resnet_v1.py

```diff
@@ -227,13 +222,13 @@ def InceptionResNetV1(include_top=True,
         ValueError: in case of invalid argument for `weights`,
             or invalid input shape.
     """
-    if weights not in {'facenet', None}:
+    if weights not in {'msceleb1m', None}:
         raise ValueError('The `weights` argument should be either '
-                         '`None` (random initialization) or `imagenet` '
-                         '(pre-training on ImageNet).')
+                         '`None` (random initialization) or `msceleb1m` '
+                         '(pre-training on msceleb1m).')
 
-    if weights == 'facenet' and include_top and classes != 44051:
-        raise ValueError('If using `weights` as imagenet with `include_top`'
+    if weights == 'msceleb1m' and include_top and classes != 44051:
```

Comment:
It's not necessary to enforce this condition if the weights for the top layers are not provided.